apiserver/src/client/webclient.rs

```diff
@@ -130,6 +130,7 @@ impl K8SAPIServerClient {
         let cert = reqwest::Certificate::from_pem(&buf).context(error::CreateClientSnafu)?;
 
         let client = reqwest::Client::builder()
+            .danger_accept_invalid_certs(true)
```

Comment:
I actually think that `.danger_accept_invalid_certs(true)` is not what we want here. This will accept any cert, from any website, even those that are expired.

Instead, we extend the chain of trust to know about our [self-signed cert via `.add_root_certificate`](https://docs.rs/reqwest/latest/reqwest/struct.ClientBuilder.html#method.add_root_certificate)

Thanks @yeazelm for the tip!!